packages/source-storybook/src/index.ts

```diff
@@ -23,7 +23,8 @@ export const schema = baseSchema.merge(
       .array(
         z.object({
           description: z.string(),
-          url: z.string(),
+          storiesUrl: z.string().url(),
+          storyUrlPrefix: z.string().url(),
```

Comment:
Should this be optional?  If not provided `storiesUrl` will be used.

Reply:
Agreed. It's probably going to be rarely needed.

Reply:
done